currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
```

Comment:
Yes, writing these tests has been very helpful in highlighting this level of problems. This will be fixed in the next homework by rewriting the architecture